delete_command.go

```diff
@@ -87,6 +92,14 @@ func deleteSnapshots(c DeleteCommand) error {
 	session := session.New(&aws.Config{Region: &c.AwsRegion})
 	svc := ec2.New(session)
 
+	if c.InstanceId == "" {
+		instanceId, err := getInstanceIdByName(c.InstanceName, svc, c.Ui)
```

Comment:
Nice that you get to share `getInstanceIdByName()` across both `create` and `delete`